packages/vitest/src/node/reporters/renderers/windowedRenderer.ts

```diff
@@ -175,6 +174,38 @@ export class WindowRenderer {
   private write(message: string, type: 'output' | 'error' = 'output') {
     (this.streams[type] as Writable['write'])(message)
   }
+
+  private addProcessExitListeners() {
```

Comment:
At this point it's ok to have this here in `windowedRenderer`. But later on it might make sense to move this to `logger.ts` instead. 🤔